src/mscorlib/src/System/IO/DriveNotFoundException.cs

```diff
@@ -18,7 +18,7 @@ namespace System.IO {
     //Thrown when trying to access a drive that is not availabe.
     [Serializable]
     [System.Runtime.InteropServices.ComVisible(true)]
-    public class DriveNotFoundException : IOException {
+    internal class DriveNotFoundException : IOException {
```

Comment:
For the error, do you mean that you happen to know that ERROR_DRIVE_INFO is only relevant to GetVolumeInformation (which is only used in corefx)?